scripts/global-food-explorer/foods.csv

```diff
@@ -97,7 +97,7 @@ meat_goose_and_guinea_fowl,"Meat, goose and guinea fowl",Goose and guinea fowl m
 meat_horse,"Meat, horse",Horse meat,Horse meat,production animals_slaughtered animal_yield,
 meat_lamb_and_mutton,"Meat, lamb and mutton",Lamb and mutton meat,Lamb and mutton meat,production animals_slaughtered animal_yield,
 meat_mule,"Meat, mule",Mule meat,Mule meat,production animals_slaughtered animal_yield,
-meat_pig,"Meat, pig",Pig meat,Pig meat,production animals_slaughtered animal_yield,
+meat_pig,"Meat, pig",Pig meat,Pig meat,production animals_slaughtered animal_yield food_balances waste,
```

Comment:
nit: to be consistent with other lines here, it should probably be 
```suggestion
meat_pig,"Meat, pig",Pig meat,Pig meat,production animals_slaughtered food_balances waste animal_yield,
```
in that order. Not sure right now whether it makes a difference in generation, but at least it's consistent then.

Reply:
Thanks Marcel. Currently the tags don't seem to always be in the same order, if you compare:
```
meat_total,"Meat, Total",Total meat,All meat,production animals_slaughtered food_balances waste animal_feed,
meat_poultry,"Meat, poultry",Poultry meat,Poultry meat,production animals_slaughtered animal_yield food_balances waste animal_feed,
```
So I guess it doesn't matter? If you think it does, I can sort them all alphabetically, for example.

Reply:
ah, in this case they actually are in the same order, no? It's just that the second line includes `animal_yield`, whereas the first one doesn't.

the order of these tags (somewhat) determines the display order in the explorer, so better to keep the order as-is and change them up as I said above.

Reply:
Ahh, I see now! all good then.
And sorry for the confusion :)